src/DbSchema/SchemaHasherMysql.php

```diff
@@ -65,6 +65,11 @@ public function hashDb(): string
             foreach ($stmt as $row) {
                 $hash = $row['dbsignature'] ?? '';
             }
+        } elseif ($this->connection instanceof \Dibi\Connection) {
```

Comment:
I am wondering, why we have this dibi connection here.
does dibi support a way to get the underlying connection out of a dibi connection, so we can handle it like any other mysql/pdo/mysqli connection?

Reply:
changed in latest commit to use `$connection->getDriver()->getResource()`